### PR TITLE
clarify connection between --debug and --log-file options

### PR DESCRIPTION
**Pull request by iliakur**
Came out of #2036

### CODE REVIEW

File: py3status/argparsers.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def _format_action_invocation(self, action):
         "--log-file",
         action="store",
         dest="log_file",
-        help="enable logging to FILE",
+        help="enable logging to FILE, this option is not set",
```
Comment (ultrabug) [inline py3status/argparsers.py:112]:
```suggestion
        help="enable logging to FILE (this option is not set by default)",
```

please?

**Comment by ultrabug** [comment]
Thanks @iliakur ! just a small comment and let's get this in

**Comment by ultrabug** [comment]
Thanks!

**Comment by iliakur** [comment]
You're welcome, thanks for maintaining this!

**Pull request merged**